JP-3628: Revise coron3 alignment step (#8643)
spacetelescope/jwst: docs/jwst/align_refs/description.rst

sub-package that is part of Stage 3 :ref:`calwebb_coron3 <calwebb_coron3>` processing.
It computes offsets between science target
images and reference PSF images, and shifts the PSF images into
alignment. This is performed on the first integration for both the science target
data and the reference PSF data. The first integration contained in the stacked PSF data
(the result of the :ref:`stack_refs <stack_refs_step>`) step is

> aligned to the first integration within a given science target exposure.

Reviewer: Revise here - shifts are computed from the first integration and applied to all the subsequent ones.

Author: thanks, docs fixed.

This results in a new product for each science target exposure that contains a stack
of individual PSF images that have been aligned to each integration in the science

> target exposure.

Reviewer 2: Might be good to add in a line stating that in-flight data show little pointing drift or PSF instability, which is why we only align the first

sub-package that is part of Stage 3 :ref:`calwebb_coron3 <calwebb_coron3>` processing.
It computes offsets between science target
images and reference PSF images, and shifts the PSF images into

> alignment. This is performed from the first integration and applied to all the

Reviewer: ```suggestion
alignment. The alignment shifts are computed from the first integration and applied to all the
```

subsequent ones for both the science target data and the reference PSF data.
Each integration contained in the stacked PSF data
(the result of the :ref:`stack_refs <stack_refs_step>`) step is
aligned to the first integration within a given science target exposure.
This results in a new product for each science target exposure that contains a stack
of individual PSF images that have been aligned to the first integration in the science
target exposure.

> The simplification of aligning to the first science integration is due to flight data
> showing minimal drifts during an observation in line-of-sight pointing, or in PSF
> properties.
> 

Reviewer: ```suggestion
Note that aligning to the first science integration is sufficient because flight data
shows that there are minimal drifts during an observation in line-of-sight pointing, or in PSF
properties.

```
